### ISSUE TITLE
Localization

### ISSUE THREAD

**Issue by carmilso** [chmouel/nextmeeting#6]
Hi,

Just wanted to know if you are planning to add localization to this package so it can manage several languages. Otherwise that is something that I could do when I have some free time.

Thanks!

**Comment by chmouel** [comment]
@carmilso that sounds good to me if you want to add it, what kind of localization are you looking for? so the software itself is in multiple languages or there is an issue with the parser?

**Comment by carmilso** [comment]
> @carmilso that sounds good to me if you want to add it, what kind of localization are you looking for? so the software itself is in multiple languages or there is an issue with the parser?

Yes, I was talking about having it in multiple languages. So for example my system is in spanish and `gcalcli` itself is already giving me information in spanish, so it would be nice if we could add this feature to nextmeeting.

I will start working on this through a PR as soon as possible.

Btw thanks for the work, it's awesome to have `gcalcli` integrated in waybar.
